From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rong Bao <rong.bao@csmantle.top>
Date: Sun, 30 Aug 2026 18:19:46 +0800
Subject: [PATCH 2/2] BACKPORT: meson.build: prefix-ize paths in conf_data

[ Upstream commit 1f17a054c981b9fc2eb6f6e7a0a7360f60bfdf2e ]
Link: https://github.com/mchehab/rasdaemon/commit/1f17a054c981b9fc2eb6f6e7a0a7360f60bfdf2e
Link: https://github.com/mchehab/rasdaemon/pull/263 ("meson.build: fix
 prefixed path handling to match Meson expectations - #263")
Signed-off-by: Rong Bao <rong.bao@csmantle.top>
---
 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/meson.build b/meson.build
index 2a7dd9db488f45aec87919bfec9d30713d3ecb80..f645b4812f19819b62db02de66d454bb9faef9b3 100644
--- a/meson.build
+++ b/meson.build
@@ -601,11 +601,11 @@ conf_data = {
     'PACKAGE' : meson.project_name(),
     'PACKAGE_VERSION' : meson.project_version(),
 
-    'prefix' : get_option('prefix'),
-    'sysconfdir' : get_option('sysconfdir'),
-    'localstatedir' : get_option('localstatedir'),
-    'sbindir' : get_option('sbindir'),
-    'SYSCONFDEFDIR' : get_option('sysconfdir') / 'sysconfig',
+    'prefix' : prefix,
+    'sysconfdir' : prefix / get_option('sysconfdir'),
+    'localstatedir' : prefix / get_option('localstatedir'),
+    'sbindir' : prefix / get_option('sbindir'),
+    'SYSCONFDEFDIR' : prefix / get_option('sysconfdir') / 'sysconfig',
 
     'META_DATE' : run_command('date', '+%d %B %Y', check: true).stdout().strip(),
     'META_ALIAS' : meson.project_name(),
-- 
2.55.0

